addon/Feeds/PocketStoriesFeed.js

```diff
@@ -52,7 +53,9 @@ module.exports = class PocketStoriesFeed extends PocketFeed {
         "url": s.dedupe_url,
         "lastVisitDate": s.published_timestamp,
         "pocket": true
-      }));
+      }))
+      .filter(s => !PlacesProvider.links.blockedURLs.has(s.url));
```

Comment:
Nit: I'd recommend to do the filtering prior to the map function. That might save some cycles for us.